app/views/courses/index.html.erb

```diff
@@ -30,15 +30,21 @@
   <p class="govuk-body-s govuk-!-margin-bottom-6">You’ll be taken to a Google Form to fill in your course details.</p>
 <% end %>
 
-<%= render partial: 'course_table', locals: { courses: @self_accredited_courses } if @self_accredited_courses %>
+<% if @self_accredited_courses %>
+  <section data-qa="courses__courses-table-section">
```

Comment:
Don't think we need to repeat `courses` here. `courses__table-section` is good enough

Reply:
Fixed in 6e36e030540e6540f009da0072eba6453f02d2ec